src/region.js

```diff
@@ -269,7 +275,11 @@ const Region = MarionetteObject.extend({
     delete this.currentView;
 
     if (!view._isDestroyed) {
-      this._removeView(view, shouldDestroy);
+      if (shouldDestroy) {
+        this.removeView(view);
```

Comment:
why not just call `destroyView` here, or put the content from `destroyView` inside `removeView` that way there's only one new public method